[1m[31mERROR   [0m root:db_helper.py:77 ‚ùå Error executing query '
                    SELECT COUNT(*) FROM source_patients
                    WHERE address LIKE '%[^a-zA-Z0-9@. -]%'
                ': ('42S22', "[42S22] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Invalid column name 'address'. (207) (SQLExecDirectW)")